HealthcareApp/src/Healthcare.Application/Abstractions/Decorators/IUserManagerDecorator.cs

```diff
@@ -8,4 +9,6 @@ public interface IUserManagerDecorator<TUser> where TUser : class
     Task<TUser> FindByEmailAsync(string email);
     Task<IList<string>> GetRolesAsync(TUser user);
     Task<IdentityResult> AddToRoleAsync(TUser user, string role);
+    Task<IQueryable<TUser>> GetAllUsersAsync();
```

Comment:
As we discussed at our class, you cannot expose IQueryable at the application layer.
Redesign the solution